### PR TITLE
chore(sidebars): l10n fixes

### PR DESCRIPTION
**Pull request by argl**
### Description

Amend l10n of sidebars with `de` entries and removal of unused entries

### Motivation

yari/rari parity

### CODE REVIEW

**Review by fiji-flo — APPROVED** [review]
:+1: from my side

File: files/sidebars/addonsidebar.yaml [hunk 1]
```diff
@@ -103,7 +119,7 @@ l10n:
     WebExtensions#Getting_started: Commencer
     WebExtensions#Concepts: Concepts
     WebExtensions#User_Interface: Interface utilisateur
-    WebExtensions#How_to: Mode d'emploi
+    WebExtensions#How_to: Mode d"emploi
```
Comment (bsmth) [inline files/sidebars/addonsidebar.yaml:122]: is that intentional?

File: files/sidebars/htmlsidebar.yaml [hunk 2]
```diff
@@ -105,6 +105,49 @@ sidebar:
...
+    HTML_tables: HTML-Tabellen
+    HTML_tables_overview: Übersicht zu HTML-Tabellen
+    HTML_table_basics: Grundlagen der HTML-Tabellen
+    HTML_table_advanced_features_and_accessibility: Erweiterte Funktionen und Zugänglichkeit von HTML-Tabellen
+    Assessment_Structuring_planet_data: "Aufgabe: Strukturierung von Planetendaten"
+    Reference: Referenzen
+    HTML_Elements: HTML-Elemente
+    Global_attributes: Globale Attribute
+    Attributes: Attribute
+    Input_types: <code>&lt;input&gt;</code>-Typen
```
Comment (bsmth) [inline files/sidebars/htmlsidebar.yaml:139]: Is HTML allowed?

File: files/sidebars/learnsidebar.yaml [hunk 3]
```diff
@@ -601,7 +641,7 @@ l10n:
     Further_resources: 更多資源
     Common_questions: 常見問題
   fr:
-    Complete_beginners_start_here: Vous débutez ? Commencez ici !
+    Complete_beginners_start_here: Vous débutezxa0? Commencez icixa0!
```
Comment (bsmth) [inline files/sidebars/learnsidebar.yaml:644]: Typo

Comment (bsmth) [reply files/sidebars/learnsidebar.yaml:644]: or search/replace problem

Comment (argl) [reply files/sidebars/learnsidebar.yaml:644]: That is a badly injured non-breaking space `0xa0` character, failing its job on french punctuation rules.

File: files/sidebars/addonsidebar.yaml (hunk 1 above)
Comment (fiji-flo) [reply files/sidebars/addonsidebar.yaml]: fmt-sidebars to the rescue.

File: files/sidebars/learnsidebar.yaml (hunk 3 above)
Comment (fiji-flo) [reply files/sidebars/learnsidebar.yaml]: replaces with &nbsp;

**Pull request merged**